src/components/Grid/GridRow.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <tr>
+    <th>
+        <span class="label">
+          {{label}}
+        </span>
+    </th>
+    <td>
+        <facet-option class="selectRow"><font-awesome-icon icon="arrow-right" /></facet-option>
+    </td>
+    <td :key="index"
+        v-for="(count,index) in counts">
+          <facet-option
+            @facetToggled="$emit('facetToggled', index)"
+            :isSelected="selected[index]"
+            class="selectItem"
+          >
+          {{formatter(count)}}
+        </facet-option>
+    </td>
+  </tr>
+</template>
+
+<script>
+import Vue from 'vue'
+import FacetOption from '../facets/FacetOption.vue'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faArrowRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+library.add(faArrowRight)
+
+export default Vue.extend({
+  props: { 
+    label: {
+      type: String,
+      required: true
+    },
+    counts: {
+      type: Array,
+      required: true
+    },
+    selected: {
+      type: Array,
+      required: true
+    }
+  },
+  components: { FacetOption, FontAwesomeIcon },
+  methods: {
+    formatter (num) {
```

Comment:
maybe select more descriptive name , or add comment
and add unit test

Reply:
the count formatter is getting moved, not introduced here